build.py

```diff
@@ -0,0 +1,263 @@
+    projGenParams.extend (['-B', str(buildPath)])
+
+    if platformName == 'WIN':
+        projGenParams.extend (['-G', 'Visual Studio 16 2019'])
+    elif platformName == 'MAC':
+        projGenParams.extend (['-G', 'Xcode'])
+
+    projGenParams.append (f'-DAC_API_DEVKIT_DIR={str(devKitFolder / "Support")}')
+    projGenParams.append (f'-DCMAKE_BUILD_TYPE={configuration}')
+
+    if languageCode is not None:
+        projGenParams.append (f'-DAC_ADDON_LANGUAGE={languageCode}')
+
+    if optionalParams is not None:
+        for key in optionalParams:
+            projGenParams.append (f'-D{key}={optionalParams[key]}')
+
+    projGenParams.append (str(rootFolder))
+
+    projGenResult = subprocess.call (projGenParams)
+    if projGenResult != 0:
+        print ('Failed to generate project')
+        return 1
+
+    # Add params to build AddOn
+    buildParams = []
+    buildParams.append ('cmake')
+    buildParams.extend (['--build', str(buildPath)])
+    buildParams.extend (['--config', configuration])
+
+    buildResult = subprocess.call (buildParams)
+    if buildResult != 0:
+        print ('Failed to build project')
+        return 1
+
+    return 0
+
+def CopyResultToPackage(packageRootFolder, buildFolder, addOnName, platformName, configuration, languageCode=None, isRelease=False):
+    packageFolder = packageRootFolder
+    sourceFolder = buildFolder / addOnName
+
+    if languageCode is not None:
+        packageFolder = packageFolder / languageCode
+        sourceFolder = sourceFolder / languageCode
+    sourceFolder = sourceFolder / configuration
+
+    if not packageFolder.exists():
+        packageFolder.mkdir(parents=True)
+
+    fileName = addOnName
+    if not isRelease:
+        fileName = f'{fileName}_{configuration}'
+
+    if platformName == 'WIN':
+        shutil.copy (
+            sourceFolder / f'{addOnName}.apx',
+            packageFolder / f'{fileName}.apx',
+        )
+        if configuration == 'Debug':
+            shutil.copy (
+            sourceFolder / f'{addOnName}.pdb',
+            packageFolder / f'{fileName}.pdb',
+        )
+
+    elif platformName == 'MAC':
+        subprocess.call ([
+            'cp', '-r',
+            sourceFolder / f'{addOnName}.bundle',
+            packageFolder / f'{fileName}.bundle'
+        ])
+
+
+
+def Main():
+    args = ParseArguments()
+
+    if args.language is None and args.release:
+        print('Must specify AddOn language on release version')
+        return 1
+
+    acVersionList = args.acVersion.split()
+
+    if args.devKitPath is not None and len(acVersionList) != 1:
+        print('Only one Archicad version supported with local APIDevKit option')
+        return 1
+
+    # Check platform operating system
+    platformName = None
+    if platform.system () == 'Windows':
+        platformName = 'WIN'
+    elif platform.system () == 'Darwin':
+        platformName = 'MAC'
+
+    # Load config data
+    configFile = open('config.json')

```

Comment:
I would make this a parameter since the build script is in the tools repository and - if I understand correctly - the config file is in the add-on repository.

Reply:
Fixed